### ISSUE TITLE
Some display issues

### ISSUE THREAD

**Issue by abcdlzy** [MicrosoftDocs/windowsserverdocs#107]
https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/diskcomp

```
Applies To: Windows Server 2016, Windows Server 2012 R2, Windows Server 2012 Compares the contents of two floppy disks. If used without parameters, diskcomp uses the current drive to compare both disks.

For examples of how to use this command, see Examples.
```

Should  display

```
Applies To: Windows Server 2016, Windows Server 2012 R2, Windows Server 2012

Compares the contents of two floppy disks. If used without parameters, diskcomp uses the current drive to compare both disks.For examples of how to use this command, see Examples.
```

**Comment by abcdlzy** [comment]
https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/chglogon

chglogon title display issus

**Comment by abcdlzy** [comment]
fixed in  5679223 and 1af43cc
